Vokal-Cocoa Touch Application Base.xctemplate/Scripts/DevsJustWantToHaveFun.sh

```diff
@@ -12,15 +12,22 @@ if [ -z "${PROJECT_NAME}" ]; then
     PROJECT_NAME="___PACKAGENAME___"
 fi
 
-# Path relative to this script where the TrueColors file you want to use to generate
+# Resources path
+RESOURCES_DIR="${PROJECT_DIR}/___PACKAGENAME___/Resources"
+
+echo "Resources dir: ${RESOURCES_DIR}"
+
+# Path where the TrueColors file you want to use to generate
 # classes is located. Must include the .truecolors extension. 
-TRUECOLORS_PATH="../Resources/___PACKAGENAME___.truecolors"
+TRUECOLORS_PATH="${RESOURCES_DIR}/___PACKAGENAME___.truecolors"
 
-# Path relative to this script where files should be output.
-OUTPUT_DIR="../Resources/TrueColors"
+echo "TRUECOLORS_PATH: ${TRUECOLORS_PATH}"
```

Comment:
Did you intend to leave this echo statement and the one on line 18 in?  If so, maybe make them consistent with one another?